docs/CHANGEMAKER_DATA.md

```diff
@@ -0,0 +1,59 @@
+# Changemaker data in the Philanthropy Data Commons (PDC)
+
+Data about Changemakers (such as grant seekers, applicants, non-profits, etc.),
+are stored and retrieved via the central PDC instance API.
+
+## Storing Changemaker data
+
+To add a Changemaker to the PDC, use `POST /changemakers`. Note, however, that
+this endpoint only registers a Changemaker by tax ID and name. It is not the
+endpoint to add other Changemaker attributes such as location or contacts. One
+writes Changemaker data to PDC via `POST /proposalVersions` and its associated
+endpoints. In other words, proposal data contains much data about Changemakers,
+therefore proposal data is an important source of Changemaker data within the
+PDC. As of this writing proposals are the sole source of Changemaker data.
```

Comment:
Right now this section is awkward because we actually just don't support the use case that it seems to be aimed at: editing / managing changemaker data in the context of that changemaker.

I would probably adjust this to just say "changemaker data cannot be directly written to the PDC, though that functionality is on the roadmap.  Instead, changemaker data is aggregated from any proposals that have been submitted by the changemaker"